examples/classifier_compression/compress_classifier.py

```diff
@@ -158,19 +161,36 @@ def main():
     if args.earlyexit_thresholds:
         msglogger.info('=> using early-exit threshold values of %s', args.earlyexit_thresholds)
 
-    # We can optionally resume from a checkpoint
-    if args.resume:
-        model, compression_scheduler, start_epoch = apputils.load_checkpoint(model, chkpt_file=args.resume)
-        model.to(args.device)
+    # TODO(barrh): args.deprecated_resume is deprecated since v0.3
+    if args.deprecated_resume:
+        msglogger.warning('The "--resume" flag is deprecated. Please use "--resume-from=YOUR_PATH" instead.')
+        if not args.reset_optimizer:
+            msglogger.warning('If you wish to also reset the optimizer, call with: --reset-optimizer')
+            args.reset_optimizer = True
+        args.resumed_checkpoint_path = args.deprecated_resume
 
-    # Define loss function (criterion) and optimizer
+    # We can optionally resume from a checkpoint
+    optimizer = None
+    if args.resumed_checkpoint_path:
+        model, compression_scheduler, optimizer, start_epoch = apputils.load_checkpoint(
+            model, args.resumed_checkpoint_path, model_device=args.device)
+    elif args.load_model_path:
+        model = apputils.load_lean_checkpoint(model, args.load_model_path,
+                                              model_device=args.device)
+    if args.reset_optimizer:
+        start_epoch = 0
+        if optimizer is not None:
+            optimizer = None
+            msglogger.info('Overriding resumed optimizer')
```

Comment:
Just to make it clearer and stand out a bit more in the log:
```suggestion
            msglogger.info()
            msglogger.info('reset_optimizer flag set: Overriding resumed optimizer and resetting epoch count to 0')
```